tests/stress_tests/memleaks_tests/tests_pipelines/tests_pipelines.cpp

```diff
@@ -144,7 +144,7 @@ TestResult test_load_unload_plugin(const std::string& target_device, const int&
 
 TestResult test_read_network(const std::string& model, const int& n) {
```

Comment:
@ilya-lavrenov please, remove `read_network` tests at all because it duplicates `create_cnnnetwork` tests

Reply:
done. But removed `create_cnnnetwork` instead